### ISSUE TITLE
Botpage

### ISSUE THREAD

**Issue by FelixGail** [BjoernPetersen/MusicBot#111]
atm only android users are able to use the bot using the [AndroidMusicBot](https://github.com/BjoernPetersen/AndroidMusicBot) and providing apple users with an app is to expensive.

A workaround would be a website provided by the machine that runs the bot.

**Comment by BjoernPetersen** [comment]
Good luck with that. Creating such a website is absolutely possible by using the same API the Android app does, so no changes on this project would be needed...

**Comment by FelixGail** [comment]
yes, isn't this the right place to note such concept for further reference?

**Comment by BjoernPetersen** [comment]
Sure
